lib/data/external/GcpClient.js

```diff
@@ -80,9 +81,34 @@ class GcpClient extends AwsClient {
                 return cb(null, bucketResp);
             });
         };
+
+        const checkBucketVersioning = (headResp, cb) => {
+            if (headResp.err) {
+                return cb(null, headResp);
+            }
+            let bucketResp;
+            return this._client.getBucketVersioning(
+            { Bucket: headResp.gcpBucket }, (err, data) => {
+                bucketResp = { gcpBucket: headResp.gcpBucket };
+                if (err) {
+                    bucketResp.error = err;
+                } else if (!data.Status || data.Status === 'Suspended') {
```

Comment:
I'm not sure but it might be possible that a status exists and is `'Disabled'`, in which case it could be useful to add it to the condition, or reverse it as `data.Status !== 'Enabled'`.